data/json/vehicleparts/roof.json

```diff
@@ -0,0 +1,45 @@
+[
+  {
+    "id": "roof_wood",
+    "type": "vehicle_part",
+    "name": "wooden roof",
+    "item": "frame_wood",
+    "color": "brown",
+    "durability": 130,
+    "breaks_into": [ { "item": "splinter", "count": [ 7, 9 ] } ],
+    "mounts": [ { "name": "on_roof", "volume": 20, "weight": 1000 } ],
```

Comment:
I assume it's a mount point restriction for things that have to stay on when going fast and won't fall off it unless the roof is wrecked.